src/main/java/org/fundacionjala/coding/manuel/Kata4.java

```diff
@@ -0,0 +1,47 @@
+package org.fundacionjala.coding.manuel;
+
+/**
+ * Created by Administrator on 3/1/2018.
+ */
+public class Kata4 {
+    /**
+     * A digital root is the recursive sum of all the digits in a number.
+     * Given n, take the sum of the digits of n. If that value has two digits,
+     * continue reducing in this way until a single-digit number is produced.
+     * This is only applicable to the natural numbers.
+     */
+
+    private static final int MAX_DIGIT = 9;
+    private static final int MOD_DIV_NUMBER = 10;
+
+    /**
+     * Default constructor.
+     */
+    public Kata4() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)